node/engine/messageservice/p2p-message-service/service.go

```diff
@@ -41,8 +40,9 @@ type peerExchangeMessage struct {
 }
 
 const (
-	PROTOCOL_ID                  protocol.ID = "/go-nitro/msg/1.0.0"
-	PEER_EXCHANGE_PROTOCOL_ID    protocol.ID = "/go-nitro/peerinfo/1.0.0"
```

Comment:
I replaced the `go-nitro` part with just `nitro` in case there are other node types in the future that are written in different languages